avocado/plugins/yaml_to_mux.py

```diff
@@ -413,9 +415,9 @@ def str_variants(self):
         out += tree_repr
         out += "\n\n"
 
-        for variant_id, tpl in self:
-            paths = ', '.join([x.path for x in tpl[0]])
-            out += 'Variant %s:    %s\n' % (variant_id, paths)
+        for variant in self:
+            paths = ', '.join([x.path for x in variant["variant"]])
+            out += 'Variant %s:    %s\n' % (variant["variant_id"], paths)
```

Comment:
There are many issues I'm fixing right now, mainly the replay/diff or sometimes job logging ones. The v1 should fix all of these...